libpromises/evalfunction.c

```diff
@@ -1586,6 +1623,44 @@ static FnCallResult FnCallGetMetaTags(EvalContext *ctx, ARG_UNUSED const Policy
         tagset = EvalContextClassTags(ctx, ref.ns, ref.name);
         ClassRefDestroy(ref);
     }
+    else if (strcmp(fp->name, "getbundlemetatags") == 0)    
+    {
+        const char *bundleref = RlistScalarValue(finalargs);
+        const Rlist *args = RlistFromSplitString(bundleref, ':');
```

Comment:
`RlistFromSplitString()` can return `NULL` if the first argument is `NULL`. This should not really happen, because `RlistScalarValue()` does not return `NULL`. However, an `assert()` would be nice, to test this assumption.